content/guides/references/migration-guide.md

```diff
@@ -366,8 +366,7 @@ required for component testing
 ```json
 {
   "experimentalComponentTesting": true,
-  "componentFolder": "src",
-  "testFiles": "**/*spec.{js,jsx,ts,tsx}"
+  "specPattern": "**/*spec.{js,jsx,ts,tsx}"
```

Comment:
Revert all changes to the Migration Guide, the `specPattern` change won't be retroactively applied to older versions of Cypress

Reply:
Yep 👍 I had been filtering out this page as to not change it. Must have slipped in there during one of the project wide searches.